twisher/src/main/java/posts/Posts.java

```diff
@@ -0,0 +1,189 @@
+                        "Однако используемый технологический процесс позволяет создавать только прямые дорожки, идущие от контакта к краю чипа без изгибов и параллельные сторонам кристалла,",
+                        dateFormat.parse("2020-01-19T13:35:00.000-0300"),
+                        "Task",
+                        "https://sun1.beltelecom-by-minsk.userapi.com/Uv7oJSk0ePo7QMQQZgWN1al2w0hF9FqcL5d11Q/MSdYzJobMBk.jpg",
+                        Collections.emptyList()
+                ),
+                new Post("2",
+                        "Однако используемый технологический процесс позволяет создавать только прямые дорожки, идущие от контакта к краю чипа без изгибов и параллельные сторонам кристалла,",
+                        dateFormat.parse("2020-01-19T13:35:00.000-0300"),
+                        "Task",
+                        "https://sun1.beltelecom-by-minsk.userapi.com/Uv7oJSk0ePo7QMQQZgWN1al2w0hF9FqcL5d11Q/MSdYzJobMBk.jpg",
+                        Collections.emptyList()
+                ),
+                new Post("3",
+                        "Для успешной распайки необходимо от каждого контакта протянуть проводящую дорожку к одной из сторон чипа для последующего закрепления на ноге интегральной схемы.",
+                        dateFormat.parse("2020-01-19T13:30:00.000-0300"),
+                        "User_1",
+                        "",
+                        Arrays.asList("User_1", "User_3")
+                ),
+                new Post("4",
+                        "Однако используемый технологический процесс позволяет создавать только прямые дорожки, идущие от контакта к краю чипа без изгибов и параллельные сторонам кристалла,",
+                        dateFormat.parse("2020-01-19T13:35:00.000-0300"),
+                        "Task",
+                        "https://sun1.beltelecom-by-minsk.userapi.com/Uv7oJSk0ePo7QMQQZgWN1al2w0hF9FqcL5d11Q/MSdYzJobMBk.jpg",
+                        Collections.emptyList()
+                )
+
+        };
+        return new Posts(Arrays.asList(arr));
+    }
+
+    public Post getPostByID(String id) {
+        for (Post post : posts) {
+            if (post.id.equals(id)) {
+                return post;
+            }
+        }
+        return null;
+    }
+    public int getID(String id) {
+        int i=0;
+        for (Post post : posts) {
+            if (post.id.equals(id)) {
+                return i;
+            }
+            i = i + 1;
+        }
+        return -1;
+    }
+    public boolean add(Post post) {
+        if (getPostByID(post.getId()) == null && post.validatePost()) {
+            posts.add(post);
+            return true;
+        }
+        return false;
+    }
+    /*public static class ForEdit {
+        String description;
+        String photoLink;
+    }*/
+    public boolean editPost(String id, ForEdit edit) {
+        if (getPostByID(id) == null) {
+            return false;
+        }
+        if (!edit.description.equals("")) {
```

Comment:
Is `description` guaranteed to be non-null? Otherwise, a NullPointerException is possible here